<?xml version="1.0"?>
<launch>
  <node pkg="pepperl_fuchs_r2000" type="r2000_node" name="r2000_driver_node" output="screen">
    <param name="scanner_ip" value="10.0.10.9"/>
    <param name="frame_id" value="scan"/>
    <param name="scan_frequency" value="35"/>
    <param name="samples_per_scan" value="3600"/>
  </node>
  <node pkg="dummy_slam_broadcaster" type="dummy_slam_broadcaster_node" name="dummy_slam_broadcaster"/>
  <node name="rvizLocal" pkg="rviz" type="rviz" respawn="false" output="screen" args="-d $(find pepperl_fuchs_r2000)/rviz/test.rviz" required="true"/>
</launch>

